x/nft/errors.go

```diff
@@ -1,16 +1,15 @@
 package nft
 
 import (
-	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"cosmossdk.io/errors"
 )
 
 // x/nft module sentinel errors
 var (
-	ErrInvalidNFT     = sdkerrors.Register(ModuleName, 2, "invalid nft")
-	ErrClassExists    = sdkerrors.Register(ModuleName, 3, "nft class already exist")
-	ErrClassNotExists = sdkerrors.Register(ModuleName, 4, "nft class does not exist")
-	ErrNFTExists      = sdkerrors.Register(ModuleName, 5, "nft already exist")
-	ErrNFTNotExists   = sdkerrors.Register(ModuleName, 6, "nft does not exist")
-	ErrInvalidID      = sdkerrors.Register(ModuleName, 7, "invalid id")
-	ErrInvalidClassID = sdkerrors.Register(ModuleName, 8, "invalid class id")
+	ErrClassExists    = errors.Register(ModuleName, 2, "nft class already exist")
```

Comment:
I don't think we can do that. The error codes shouldn't change, but you can add new errors codes.

Reply:
if we accept this, it would mean that we cant release this in a minor release as it may cause consensus issues

Reply:
> if we accept this, it would mean that we cant release this in a minor release as it may cause consensus issues

I think removing the regex verification of classID and nftID is already a break change. So if you just keep the error code unchanged, nothing seems to be the same